message.go

```diff
@@ -38,6 +38,10 @@ type PubAddr struct {
 	OutPrice string `json:"outPrice,omitempty"`
 }
 
+func (pa *PubAddr) String() string {
```

Comment:
It's a convention to put method after struct definition

Reply:
Yes, moved them after struct definition.

Reply:
Yes. It is after the struct definition.